packages/components/buttons/primary-button/index.js

```diff
@@ -0,0 +1 @@
+export { default } from '../../../../src/components/buttons/primary-button';
```

Comment:
Can we try having a flat structure for the packages? So that every package is under `packages/*`

Reply:
We can also have other top level workspace folders, like:

```
packages/
presets/
utils/
```

and so on

Reply:
I left it like this so that it matches the current src structure, which made it a lot easier to ensure I didn't miss any packages...

Reply:
Ok, I guess we can reorganize things as a next step